tests/Unit/ControlSystem/Test_FunctionOfTime.cpp

```diff
@@ -0,0 +1,84 @@
+// Distributed under the MIT License.
```

Comment:
Sorry, I was not clear enough in our last comment. What we had in mind was moving these tests into the Test_PiecewisePolynomial.cpp file and not having a separate file for the base class at all. The reason is that you can't really test a pure virtual class since all it does is create the vtable to allow for an interface.